proc-macros/tests/ui/correct/basic.rs

```diff
@@ -111,6 +122,17 @@ async fn main() {
 	let client = WsClientBuilder::default().build(&server_url).await.unwrap();
 
 	assert_eq!(client.async_method(10, "a".into()).await.unwrap(), 42);
+
+	let mut params = ObjectParams::new();
+	params.insert("type", 256).unwrap();
+	params.insert("halfType", true).unwrap();
+	assert_eq!(client.request::<u16, ObjectParams>("foo_rename_params", params).await.unwrap(), 128);
```

Comment:
can you add another call here with ArrayParams to ensure that the rename stuff doesn't break it?

Reply:
added